Gemfile

```diff
@@ -8,6 +8,13 @@ group :development do
   gem 'pry'
 end
 
+if RUBY_VERSION < "2.0"
+  gem 'json', '< 2.0.0'
+  gem 'net-ssh', '<= 2.10'
+  gem 'rest-client', '< 1.7'
```

Comment:
`< 2` doesn't ensure anything, it allows ancient versions to be used. `~> 1.8` prevents at least a few issues and we know it's compatible with it